sql/core/src/main/scala/org/apache/spark/sql/execution/joins/BroadcastLeftSemiJoinHash.scala

```diff
@@ -70,7 +77,10 @@ case class BroadcastLeftSemiJoinHash(
               InternalAccumulator.PEAK_EXECUTION_MEMORY).add(unsafe.getUnsafeSize)
           case _ =>
         }
-        hashSemiJoin(streamIter, numLeftRows, hashedRelation, numOutputRows)
+        sj match {
+          case LeftSemi => hashSemiJoin(streamIter, numLeftRows, hashedRelation, numOutputRows)
+          case LeftAnti => hashAntiJoin(streamIter, numLeftRows, hashedRelation, numOutputRows)
```

Comment:
OK, but leave it for now until we feel it's ready to be merge, otherwise, it probably conflict-prone.